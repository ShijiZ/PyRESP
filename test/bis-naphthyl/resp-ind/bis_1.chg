All values are reported in atomic units
%FLAG TITLE
 molecule 1: naphthyl

%FLAG ATOM CRD: I4,3E16.7
 atm.no      X               Y               Z
   1  -3.4196870E-01  -3.8245240E-01  -2.2379470E-01
   2  -3.0137120E+00  -6.6733250E-01  -1.7237140E-01
   3  -4.1655650E+00  -3.0774580E+00  -4.5241950E-01
   4  -6.7218550E+00  -3.3337600E+00  -4.0980620E-01
   5  -8.2871570E+00  -1.1876780E+00  -8.2104850E-02
   6  -7.2380090E+00   1.1409360E+00   1.9545440E-01
   7  -4.5750250E+00   1.4616850E+00   1.5977450E-01
   8  -3.3997450E+00   3.8559500E+00   4.4761770E-01
   9  -8.4462660E-01   4.0664900E+00   3.9297250E-01
  10   7.6098920E-01   1.9305880E+00   4.8666140E-02
  11   8.0949660E-01  -2.0403480E+00  -4.5754250E-01
  12  -2.9636740E+00  -4.6991780E+00  -6.9257350E-01
  13  -7.5698620E+00  -5.1703360E+00  -6.2116640E-01
  14  -1.0308890E+01  -1.4128860E+00  -5.1018840E-02
  15  -8.4208230E+00   2.7785030E+00   4.4958490E-01
  16  -4.5649300E+00   5.5057230E+00   7.0573740E-01
  17   1.3078800E-02   5.9002260E+00   6.1156470E-01
  18   3.5994910E+00   2.3574250E+00   2.7072230E-02
  19   5.2326990E+00   2.5578210E-01  -1.1285470E+00
  20   7.8846260E+00   1.2812330E+00  -1.6756280E+00
  21   5.4574950E+00  -2.0694230E+00   6.7427150E-01
  22   4.6853510E+00  -4.1020210E+00  -1.7990010E-01
  23   6.3981970E+00  -1.6028000E+00   2.7540580E+00
  24   4.3903760E+00  -3.6916190E-01  -2.8929710E+00
  25   9.0685060E+00  -1.8308430E-01  -2.4959300E+00
  26   8.7740430E+00   1.9007740E+00   6.2340200E-02
  27   7.8358690E+00   2.8633100E+00  -2.9949260E+00
  28   4.2425500E+00   2.6484730E+00   1.9549130E+00
  29   3.9400330E+00   4.1246770E+00  -9.7823600E-01

%FLAG ATOM CHRG: 2(I4,X7),I4,X2,E16.7
 atm.no   element.no   ivary      q(opt)
   1          6          0    -1.0345539E-01
   2          6          0    -5.3475632E-02
   3          6          0    -1.5351308E-01
   4          6          0    -4.0561954E-01
   5          6          0    -3.8730366E-01
   6          6          0    -1.4972817E-01
   7          6          0    -6.4073629E-02
   8          6          0    -3.1055385E-01
   9          6          0    -2.9000868E-01
  10          6          0    -8.3607568E-02
  11          1          0     1.7371743E-01
  12          1          0     2.6524454E-01
  13          1          0     2.9213974E-01
  14          1          0     2.9645324E-01
  15          1          0     2.7912620E-01
  16          1          0     2.6951498E-01
  17          1          0     2.5060711E-01
  18          6          0    -2.2707977E-03
  19          6          0     4.2979259E-01
  20          6          0    -2.1519803E-01
  21          6          0     3.9635359E-01
  22          8          0    -8.7065257E-01
  23          8         22    -8.7065257E-01
  24          1          0    -3.0104154E-02
  25          1          0     7.2416352E-02
  26          1          0     4.4416147E-02
  27          1          0     7.0997086E-02
  28          1          0     5.8334376E-02
  29          1          0     9.1103931E-02

%FLAG IND DIP GLOBAL: I4,3E16.7
 atm.no      X               Y               Z
   1   6.4687396E-03   4.2136459E-02   3.4937318E-02
   2   4.5094952E-02  -1.2855254E-02  -3.6817576E-03
   3  -2.3268266E-01   2.0303672E-01   4.1212609E-02
   4   7.7637075E-02   2.8693250E-01   3.6868161E-02
   5   3.3718305E-01  -9.5313242E-02  -2.0804415E-02
   6   1.8667113E-01  -3.7095735E-01  -5.2722607E-02
   7   7.6720105E-02   9.7980524E-02   1.3361648E-02
   8   2.3232937E-01  -2.3468102E-01  -3.6401512E-02
   9  -1.3842517E-01  -2.3338449E-01  -2.3257912E-02
  10  -1.6121045E-01   4.3264441E-03   2.8032089E-02
  11   6.6597431E-02  -2.1237555E-02   1.8138422E-02
  12  -9.0974411E-03   5.4281459E-02   1.1224378E-02
  13   4.3797803E-02   7.9815418E-02   9.9467425E-03
  14   9.4974830E-02   9.3517770E-03  -1.1137843E-03
  15   5.7664999E-02  -6.3574273E-02  -9.6193079E-03
  16   4.7785925E-02  -7.4930738E-02  -1.0594834E-02
  17  -2.3472629E-02  -7.5181729E-02  -6.0759655E-03
  18  -3.1672879E-02  -2.3375183E-01   7.8196558E-02
  19  -3.6343307E-03  -3.8981712E-01   2.6197107E-01
  20  -9.2353180E-02  -1.7474450E-01   1.6831349E-01
  21   8.4039832E-02  -6.4058633E-01   5.0157659E-01
  22   7.3206520E-03  -8.3985960E-02  -3.2101699E-03
  23   3.0543099E-02  -2.4589615E-02   8.0934179E-02
  24   2.4166916E-03  -4.7691257E-02   2.5968572E-02
  25  -1.4383681E-02  -9.4489762E-03   1.9609611E-02
  26  -1.5180408E-02  -2.9209532E-02   2.6973935E-02
  27  -1.4779667E-03  -1.5320668E-02   1.4787202E-02
  28   7.4022789E-03  -4.8093723E-02   1.8824141E-02
  29  -1.4907967E-02  -2.4125252E-02   7.0496576E-03

%FLAG TITLE
 molecule 2: naphthyl

%FLAG ATOM CRD: I4,3E16.7
 atm.no      X               Y               Z
  30  -3.4196870E-01  -3.8245240E-01  -2.2379470E-01
  31  -3.0137120E+00  -6.6733250E-01  -1.7237140E-01
  32  -4.1655650E+00  -3.0774580E+00  -4.5241950E-01
  33  -6.7218550E+00  -3.3337600E+00  -4.0980620E-01
  34  -8.2871570E+00  -1.1876780E+00  -8.2104850E-02
  35  -7.2380090E+00   1.1409360E+00   1.9545440E-01
  36  -4.5750250E+00   1.4616850E+00   1.5977450E-01
  37  -3.3997450E+00   3.8559500E+00   4.4761770E-01
  38  -8.4462660E-01   4.0664900E+00   3.9297250E-01
  39   7.6098920E-01   1.9305880E+00   4.8666140E-02
  40   8.0949660E-01  -2.0403480E+00  -4.5754250E-01
  41  -2.9636740E+00  -4.6991780E+00  -6.9257350E-01
  42  -7.5698620E+00  -5.1703360E+00  -6.2116640E-01
  43  -1.0308890E+01  -1.4128860E+00  -5.1018840E-02
  44  -8.4208230E+00   2.7785030E+00   4.4958490E-01
  45  -4.5649300E+00   5.5057230E+00   7.0573740E-01
  46   1.3078800E-02   5.9002260E+00   6.1156470E-01
  47   3.5994910E+00   2.3574250E+00   2.7072230E-02
  48   5.2326990E+00   2.5578210E-01  -1.1285470E+00
  49   7.8846260E+00   1.2812330E+00  -1.6756280E+00
  50   5.4574950E+00  -2.0694230E+00   6.7427150E-01
  51   4.6853510E+00  -4.1020210E+00  -1.7990010E-01
  52   6.3981970E+00  -1.6028000E+00   2.7540580E+00
  53   4.3903760E+00  -3.6916190E-01  -2.8929710E+00
  54   9.0685060E+00  -1.8308430E-01  -2.4959300E+00
  55   8.7740430E+00   1.9007740E+00   6.2340200E-02
  56   7.8358690E+00   2.8633100E+00  -2.9949260E+00
  57   4.2425500E+00   2.6484730E+00   1.9549130E+00
  58   3.9400330E+00   4.1246770E+00  -9.7823600E-01

%FLAG ATOM CHRG: 2(I4,X7),I4,X2,E16.7
 atm.no   element.no   ivary      q(opt)
  30          6          1    -1.0345539E-01
  31          6          2    -5.3475632E-02
  32          6          3    -1.5351308E-01
  33          6          4    -4.0561954E-01
  34          6          5    -3.8730366E-01
  35          6          6    -1.4972817E-01
  36          6          7    -6.4073629E-02
  37          6          8    -3.1055385E-01
  38          6          9    -2.9000868E-01
  39          6         10    -8.3607568E-02
  40          1         11     1.7371743E-01
  41          1         12     2.6524454E-01
  42          1         13     2.9213974E-01
  43          1         14     2.9645324E-01
  44          1         15     2.7912620E-01
  45          1         16     2.6951498E-01
  46          1         17     2.5060711E-01
  47          6         18    -2.2707977E-03
  48          6         19     4.2979259E-01
  49          6         20    -2.1519803E-01
  50          6         21     3.9635359E-01
  51          8         22    -8.7065257E-01
  52          8         22    -8.7065257E-01
  53          1         24    -3.0104154E-02
  54          1         25     7.2416352E-02
  55          1         26     4.4416147E-02
  56          1         27     7.0997086E-02
  57          1         28     5.8334376E-02
  58          1         29     9.1103931E-02

%FLAG IND DIP GLOBAL: I4,3E16.7
 atm.no      X               Y               Z
  30   6.4687396E-03   4.2136459E-02   3.4937318E-02
  31   4.5094952E-02  -1.2855254E-02  -3.6817576E-03
  32  -2.3268266E-01   2.0303672E-01   4.1212609E-02
  33   7.7637075E-02   2.8693250E-01   3.6868161E-02
  34   3.3718305E-01  -9.5313242E-02  -2.0804415E-02
  35   1.8667113E-01  -3.7095735E-01  -5.2722607E-02
  36   7.6720105E-02   9.7980524E-02   1.3361648E-02
  37   2.3232937E-01  -2.3468102E-01  -3.6401512E-02
  38  -1.3842517E-01  -2.3338449E-01  -2.3257912E-02
  39  -1.6121045E-01   4.3264441E-03   2.8032089E-02
  40   6.6597431E-02  -2.1237555E-02   1.8138422E-02
  41  -9.0974411E-03   5.4281459E-02   1.1224378E-02
  42   4.3797803E-02   7.9815418E-02   9.9467425E-03
  43   9.4974830E-02   9.3517770E-03  -1.1137843E-03
  44   5.7664999E-02  -6.3574273E-02  -9.6193079E-03
  45   4.7785925E-02  -7.4930738E-02  -1.0594834E-02
  46  -2.3472629E-02  -7.5181729E-02  -6.0759655E-03
  47  -3.1672879E-02  -2.3375183E-01   7.8196558E-02
  48  -3.6343307E-03  -3.8981712E-01   2.6197107E-01
  49  -9.2353180E-02  -1.7474450E-01   1.6831349E-01
  50   8.4039832E-02  -6.4058633E-01   5.0157659E-01
  51   7.3206520E-03  -8.3985960E-02  -3.2101699E-03
  52   3.0543099E-02  -2.4589615E-02   8.0934179E-02
  53   2.4166916E-03  -4.7691257E-02   2.5968572E-02
  54  -1.4383681E-02  -9.4489762E-03   1.9609611E-02
  55  -1.5180408E-02  -2.9209532E-02   2.6973935E-02
  56  -1.4779667E-03  -1.5320668E-02   1.4787202E-02
  57   7.4022789E-03  -4.8093723E-02   1.8824141E-02
  58  -1.4907967E-02  -2.4125252E-02   7.0496576E-03

