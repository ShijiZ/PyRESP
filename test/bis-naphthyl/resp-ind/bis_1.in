test for bis-naphthyl
 &cntrl 
 nmol = 2,
 iqopt = 1,
 ihfree = 1,
 qwt = 0.0005,
 ioutopt = 1,
 ipol = 5,
 igdm = 1,
 exc12 = 0,
 exc13 = 0,
 ipermdip = 0,
 &end 
    1.0
naphthyl
   -1   29
    6    0
    6    0
    6    0
    6    0
    6    0
    6    0
    6    0 
    6    0 
    6    0   
    6    0 
    1    0 
    1    0 
    1    0 
    1    0 
    1    0   
    1    0 
    1    0 
    6    0 
    6    0 
    6    0 
    6    0   
    8    0 
    8   22 
    1    0 
    1    0 
    1    0  
    1    0 
    1    0 
    1    0 

    1.0
naphthyl
   -1   29
    6    0
    6    0
    6    0
    6    0
    6    0
    6    0
    6    0 
    6    0 
    6    0   
    6    0 
    1    0 
    1    0 
    1    0 
    1    0 
    1    0   
    1    0 
    1    0 
    6    0 
    6    0 
    6    0 
    6    0   
    8    0 
    8   22 
    1    0 
    1    0 
    1    0  
    1    0 
    1    0 
    1    0 

   13  -1.000
    1   19    1   20    1   21    1   22    1   23    1   24    1   25    1   26
    1   27    2   20    2   25    2   26    2   27

    2
    1    1    2    1
    2
    1    2    2    2
    2
    1    3    2    3
    2
    1    4    2    4
    2
    1    5    2    5
    2
    1    6    2    6
    2
    1    7    2    7
    2
    1    8    2    8
    2
    1    9    2    9
    2
    1   10    2   10
    2
    1   11    2   11
    2
    1   12    2   12
    2
    1   13    2   13
    2
    1   14    2   14
    2
    1   15    2   15
    2
    1   16    2   16
    2
    1   17    2   17
    2
    1   18    2   18
    2
    1   19    2   19
    2
    1   20    2   20
    2
    1   21    2   21
    2
    1   22    2   22
    2
    1   23    2   23
    2
    1   24    2   24
    2
    1   25    2   25
    2
    1   26    2   26
    2
    1   27    2   27
    2
    1   28    2   28
    2
    1   29    2   29

