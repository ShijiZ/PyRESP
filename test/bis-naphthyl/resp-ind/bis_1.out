
 -----------------------------------------------
              Py_RESP Version 1.0
 -----------------------------------------------
 test for bis-naphthyl
 -----------------------------------------------

 ------------------
 Control Parameters
 ------------------
 nmol        = 2   iqopt       = 1
 ihfree      = 1   irstrnt     = 1
 ioutopt     = 1   ireornt     = 0
 iquad       = 1   qwt         = 0.00050000
 ipol        = 5   igdm        = 1
 exc12       = 0   exc13       = 0

 --------------------
 Molecule Information
 --------------------
 Multiple-molecule run of 2 molecules

 Molecule 1 weight:     1.000
 Molecule 1 Name: naphthyl
 Total charge (ich): -1
 Number of centers: 29
    1    6    0
    2    6    0
    3    6    0
    4    6    0
    5    6    0
    6    6    0
    7    6    0
    8    6    0
    9    6    0
   10    6    0
   11    1    0
   12    1    0
   13    1    0
   14    1    0
   15    1    0
   16    1    0
   17    1    0
   18    6    0
   19    6    0
   20    6    0
   21    6    0
   22    8    0
   23    8   22
   24    1    0
   25    1    0
   26    1    0
   27    1    0
   28    1    0
   29    1    0

 Molecule 2 weight:     1.000
 Molecule 2 Name: naphthyl
 Total charge (ich): -1
 Number of centers: 29
   30    6    0
   31    6    0
   32    6    0
   33    6    0
   34    6    0
   35    6    0
   36    6    0
   37    6    0
   38    6    0
   39    6    0
   40    1    0
   41    1    0
   42    1    0
   43    1    0
   44    1    0
   45    1    0
   46    1    0
   47    6    0
   48    6    0
   49    6    0
   50    6    0
   51    8    0
   52    8   22
   53    1    0
   54    1    0
   55    1    0
   56    1    0
   57    1    0
   58    1    0
 ---------------------------------------------------------
 Inter-Molecular Charge Constraint(s) Info
 ---------------------------------------------------------
    1   19    1   20    1   21    1   22    1   23    1   24    1   25    1   26
    1   27    2   20    2   25    2   26    2   27

 -------------------------------
 Charges Equivalence Information 
 -------------------------------
    1    1    2    1
    1    2    2    2
    1    3    2    3
    1    4    2    4
    1    5    2    5
    1    6    2    6
    1    7    2    7
    1    8    2    8
    1    9    2    9
    1   10    2   10
    1   11    2   11
    1   12    2   12
    1   13    2   13
    1   14    2   14
    1   15    2   15
    1   16    2   16
    1   17    2   17
    1   18    2   18
    1   19    2   19
    1   20    2   20
    1   21    2   21
    1   22    2   22
    1   23    2   23
    1   24    2   24
    1   25    2   25
    1   26    2   26
    1   27    2   27
    1   28    2   28
    1   29    2   29

 -----------------------------
 Atom Ivary Information
 -----------------------------
    6    0
    6    0
    6    0
    6    0
    6    0
    6    0
    6    0
    6    0
    6    0
    6    0
    1    0
    1    0
    1    0
    1    0
    1    0
    1    0
    1    0
    6    0
    6    0
    6    0
    6    0
    8    0
    8   22
    1    0
    1    0
    1    0
    1    0
    1    0
    1    0

    6    1
    6    2
    6    3
    6    4
    6    5
    6    6
    6    7
    6    8
    6    9
    6   10
    1   11
    1   12
    1   13
    1   14
    1   15
    1   16
    1   17
    6   18
    6   19
    6   20
    6   21
    8   22
    8   22
    1   24
    1   25
    1   26
    1   27
    1   28
    1   29

 -------------------------
 Input Information Summary
 -------------------------
 Total number of atoms      =   58
 Weight factor on initial charge restraints=  0.000500
 There are  3 charge constraints

 Read polarizability and radii information from:
 ../../../polarizability/pGM-pol-2016-09-01
 Assume the units are in a.u.

 --------------------------------------
 Atomic and ESP Coordinates Information
 --------------------------------------
 Reading esp's for molecule   1
 total number of atoms      =    29
 total number of esp points =  1250

 center       X               Y               Z
    1  -3.4196870E-01  -3.8245240E-01  -2.2379470E-01
    2  -3.0137120E+00  -6.6733250E-01  -1.7237140E-01
    3  -4.1655650E+00  -3.0774580E+00  -4.5241950E-01
    4  -6.7218550E+00  -3.3337600E+00  -4.0980620E-01
    5  -8.2871570E+00  -1.1876780E+00  -8.2104850E-02
    6  -7.2380090E+00   1.1409360E+00   1.9545440E-01
    7  -4.5750250E+00   1.4616850E+00   1.5977450E-01
    8  -3.3997450E+00   3.8559500E+00   4.4761770E-01
    9  -8.4462660E-01   4.0664900E+00   3.9297250E-01
   10   7.6098920E-01   1.9305880E+00   4.8666140E-02
   11   8.0949660E-01  -2.0403480E+00  -4.5754250E-01
   12  -2.9636740E+00  -4.6991780E+00  -6.9257350E-01
   13  -7.5698620E+00  -5.1703360E+00  -6.2116640E-01
   14  -1.0308890E+01  -1.4128860E+00  -5.1018840E-02
   15  -8.4208230E+00   2.7785030E+00   4.4958490E-01
   16  -4.5649300E+00   5.5057230E+00   7.0573740E-01
   17   1.3078800E-02   5.9002260E+00   6.1156470E-01
   18   3.5994910E+00   2.3574250E+00   2.7072230E-02
   19   5.2326990E+00   2.5578210E-01  -1.1285470E+00
   20   7.8846260E+00   1.2812330E+00  -1.6756280E+00
   21   5.4574950E+00  -2.0694230E+00   6.7427150E-01
   22   4.6853510E+00  -4.1020210E+00  -1.7990010E-01
   23   6.3981970E+00  -1.6028000E+00   2.7540580E+00
   24   4.3903760E+00  -3.6916190E-01  -2.8929710E+00
   25   9.0685060E+00  -1.8308430E-01  -2.4959300E+00
   26   8.7740430E+00   1.9007740E+00   6.2340200E-02
   27   7.8358690E+00   2.8633100E+00  -2.9949260E+00
   28   4.2425500E+00   2.6484730E+00   1.9549130E+00
   29   3.9400330E+00   4.1246770E+00  -9.7823600E-01


 Reading esp's for molecule   2
 total number of atoms      =    29
 total number of esp points =  1250

 center       X               Y               Z
    1  -3.4196870E-01  -3.8245240E-01  -2.2379470E-01
    2  -3.0137120E+00  -6.6733250E-01  -1.7237140E-01
    3  -4.1655650E+00  -3.0774580E+00  -4.5241950E-01
    4  -6.7218550E+00  -3.3337600E+00  -4.0980620E-01
    5  -8.2871570E+00  -1.1876780E+00  -8.2104850E-02
    6  -7.2380090E+00   1.1409360E+00   1.9545440E-01
    7  -4.5750250E+00   1.4616850E+00   1.5977450E-01
    8  -3.3997450E+00   3.8559500E+00   4.4761770E-01
    9  -8.4462660E-01   4.0664900E+00   3.9297250E-01
   10   7.6098920E-01   1.9305880E+00   4.8666140E-02
   11   8.0949660E-01  -2.0403480E+00  -4.5754250E-01
   12  -2.9636740E+00  -4.6991780E+00  -6.9257350E-01
   13  -7.5698620E+00  -5.1703360E+00  -6.2116640E-01
   14  -1.0308890E+01  -1.4128860E+00  -5.1018840E-02
   15  -8.4208230E+00   2.7785030E+00   4.4958490E-01
   16  -4.5649300E+00   5.5057230E+00   7.0573740E-01
   17   1.3078800E-02   5.9002260E+00   6.1156470E-01
   18   3.5994910E+00   2.3574250E+00   2.7072230E-02
   19   5.2326990E+00   2.5578210E-01  -1.1285470E+00
   20   7.8846260E+00   1.2812330E+00  -1.6756280E+00
   21   5.4574950E+00  -2.0694230E+00   6.7427150E-01
   22   4.6853510E+00  -4.1020210E+00  -1.7990010E-01
   23   6.3981970E+00  -1.6028000E+00   2.7540580E+00
   24   4.3903760E+00  -3.6916190E-01  -2.8929710E+00
   25   9.0685060E+00  -1.8308430E-01  -2.4959300E+00
   26   8.7740430E+00   1.9007740E+00   6.2340200E-02
   27   7.8358690E+00   2.8633100E+00  -2.9949260E+00
   28   4.2425500E+00   2.6484730E+00   1.9549130E+00
   29   3.9400330E+00   4.1246770E+00  -9.7823600E-01


 ------------------------
 Optimization Information
 ------------------------
 iqopt=1, all q0 values will be set to 0

 Number of unique UNfrozen charge centers =    28

 Non-linear optimization requested.
  qchnge =    4.6318094392E-02
  qchnge =    5.0865259083E-03
  qchnge =    1.3900724201E-03
  qchnge =    4.8557892377E-04
  qchnge =    1.8491980143E-04
  qchnge =    7.6065991199E-05
  qchnge =    3.4283156809E-05
  qchnge =    1.7038072926E-05
  qchnge =    9.2244576482E-06
  qchnge =    5.3201040502E-06
  qchnge =    3.2008558714E-06
  qchnge =    1.9789471701E-06
  qchnge =    1.2450269149E-06
  qchnge =    7.9221272770E-07

 Convergence in   13 iterations

 -----------------------
 Fitting Results Summary
 -----------------------
 molecule 1: naphthyl
 molecule 2: naphthyl

          Point Charges Before & After Optimization
 atm.no   element.no   q(init)        q(opt)     ivary    d(rstr)/dq
    1         6        0.000000     -0.103455      0       0.003475
    2         6        0.000000     -0.053476      0       0.004409
    3         6        0.000000     -0.153513      0       0.002729
    4         6        0.000000     -0.405620      0       0.001197
    5         6        0.000000     -0.387304      0       0.001250
    6         6        0.000000     -0.149728      0       0.002777
    7         6        0.000000     -0.064074      0       0.004210
    8         6        0.000000     -0.310554      0       0.001533
    9         6        0.000000     -0.290009      0       0.001630
   10         6        0.000000     -0.083608      0       0.003836
   11         1        0.000000      0.173717      0       0.000000
   12         1        0.000000      0.265245      0       0.000000
   13         1        0.000000      0.292140      0       0.000000
   14         1        0.000000      0.296453      0       0.000000
   15         1        0.000000      0.279126      0       0.000000
   16         1        0.000000      0.269515      0       0.000000
   17         1        0.000000      0.250607      0       0.000000
   18         6        0.000000     -0.002271      0       0.004999
   19         6        0.000000      0.429793      0       0.001133
   20         6        0.000000     -0.215198      0       0.002107
   21         6        0.000000      0.396354      0       0.001223
   22         8        0.000000     -0.870653      0       0.000571
   23         8        0.000000     -0.870653     22       0.000571
   24         1        0.000000     -0.030104      0       0.000000
   25         1        0.000000      0.072416      0       0.000000
   26         1        0.000000      0.044416      0       0.000000
   27         1        0.000000      0.070997      0       0.000000
   28         1        0.000000      0.058334      0       0.000000
   29         1        0.000000      0.091104      0       0.000000

   30         6        0.000000     -0.103455      1       0.003475
   31         6        0.000000     -0.053476      2       0.004409
   32         6        0.000000     -0.153513      3       0.002729
   33         6        0.000000     -0.405620      4       0.001197
   34         6        0.000000     -0.387304      5       0.001250
   35         6        0.000000     -0.149728      6       0.002777
   36         6        0.000000     -0.064074      7       0.004210
   37         6        0.000000     -0.310554      8       0.001533
   38         6        0.000000     -0.290009      9       0.001630
   39         6        0.000000     -0.083608     10       0.003836
   40         1        0.000000      0.173717     11       0.000000
   41         1        0.000000      0.265245     12       0.000000
   42         1        0.000000      0.292140     13       0.000000
   43         1        0.000000      0.296453     14       0.000000
   44         1        0.000000      0.279126     15       0.000000
   45         1        0.000000      0.269515     16       0.000000
   46         1        0.000000      0.250607     17       0.000000
   47         6        0.000000     -0.002271     18       0.004999
   48         6        0.000000      0.429793     19       0.001133
   49         6        0.000000     -0.215198     20       0.002107
   50         6        0.000000      0.396354     21       0.001223
   51         8        0.000000     -0.870653     22       0.000571
   52         8        0.000000     -0.870653     22       0.000571
   53         1        0.000000     -0.030104     24       0.000000
   54         1        0.000000      0.072416     25       0.000000
   55         1        0.000000      0.044416     26       0.000000
   56         1        0.000000      0.070997     27       0.000000
   57         1        0.000000      0.058334     28       0.000000
   58         1        0.000000      0.091104     29       0.000000

 Sum over the calculated charges:     -2.000

 --------------------------
 Fitting Statistics Summary
 --------------------------
 The initial sum of squares  (ssvpot = sum_i(espi**2))          3.2877896E+01
 The residual sum of squares (RSS = sum_i((espi-espclci)**2))   9.1395825E-03
 The root-mean-squared error (RMSE = sqrt(RSS/N))               1.9120233E-03
 The relative RMSE           (RRMSE = sqrt(RSS/ssvpot))         1.6672901E-02

 ---------------------------
 Molecular Multipole Summary
 ---------------------------
 Dipole Moments (Debye):
 #MOL         D          Dx         Dy         Dz
   1       12.89046  -10.93974    6.17869   -2.88269
 #MOL         D          Dx         Dy         Dz
   2       12.89046  -10.93974    6.17869   -2.88269

 Traceless Quadrupole Moments (Debye*Angst.):
 #MOL          X          Y          Z
   1    X  -21.10585
        Y   17.66938   10.99427
        Z  -11.66763    4.29012   10.11158
 #MOL          X          Y          Z
   2    X  -21.10585
        Y   17.66938   10.99427
        Z  -11.66763    4.29012   10.11158

 Traceless Quadrupole Moments in Principal Axes (Debye*Angst.):
 #MOL          X          Y          Z
   1    X   18.92899
        Y    0.00000   13.60720
        Z    0.00000    0.00000  -32.53619
 #MOL          X          Y          Z
   2    X   18.92899
        Y    0.00000   13.60720
        Z    0.00000    0.00000  -32.53619